target-app/Home/Views/HomeViewController.swift

```diff
@@ -62,6 +69,23 @@ class HomeViewController: UIViewController, MKMapViewDelegate {
         if let coordinates = mapView.userLocation.location?.coordinate {
             mapView.setCenter(coordinates, animated: true)
         }
+        
+        let longTapGesture = UILongPressGestureRecognizer(target: self, action: #selector(longTap))
+        mapView.addGestureRecognizer(longTapGesture)
+    }
+    
+    // MARK: - ACTIONS
+    
+    @objc func longTap(sender: UIGestureRecognizer){
+        if sender.state == .began {
+            let locationInView = sender.location(in: mapView)
+            let locationOnMap = mapView.convert(locationInView, toCoordinateFrom: mapView)
+            let location = CLLocation(latitude: locationOnMap.latitude, longitude: locationOnMap.longitude)
+            
+            mapView.removeAnnotations([mapView.annotations].last!)
```

Comment:
Pls avoid force unwrapping, since it could make the app crash.
```suggestion
            mapView.removeAnnotations(mapView.annotations)
```

for now, since you have not targets, you can remove all annotations.